examples/mcp_servers/authorization/src/authorization/.env.example

```diff
@@ -0,0 +1,16 @@
+# Server Auth environment variables
+MCP_RESOURCE_SERVER_CANONICAL_URL="http://127.0.0.1:8000/mcp"
+MCP_RESOURCE_SERVER_AUTHORIZATION_SERVERS='[
+  {
+    "authorization_server_url": "https://your-workos.authkit.app",
+    "issuer": "https://your-workos.authkit.app",
+    "jwks_uri": "https://your-workos.authkit.app/oauth2/jwks",
+    "algorithm": "RS256",
+    "verify_options": {
+      "verify_aud": false
+    }
+  }
+]'
```

Comment:
DevEx q: can I also specify AS and AS validation options in code, instead of env vars if I wanted to?